packages/api-queries/src/site-users.ts

```diff
@@ -24,3 +29,9 @@ export const siteUsersQuery = ( siteId: number ) =>
 		queryKey: [ 'site', siteId, 'users', 'list' ],
 		queryFn: () => fetchSiteUsers( siteId ),
 	} );
+
+export const siteUsersWpcomQuery = ( siteId: number, role?: string ) =>
+	queryOptions( {
+		queryKey: [ 'site', siteId, 'users', 'wpcom', role ],
```

Comment:
I'd go with a query key like

```suggestion
		queryKey: [ 'site', siteId, 'wpcom-users', role ],
```

It looks like "users" and "wpcom users" are different entities, and so we wouldn't want something like `queryClient.invalidateQueries( { queryKey: [ 'site', siteId, 'users' ] } )` to also invalidate the wpcom users, since they're unrelated.